tests/core/test_common_metadata.py

```diff
@@ -56,6 +58,7 @@ def test_store_schema_metadata(store, df_all_types):
         pa.field("byte", pa.binary()),
         pa.field("date", pa.date32()),
         pa.field("datetime64", pa.timestamp("us")),
```

Comment:
Any idea why this is `us`? Shouldn't the default accuracy be `ns`?? cc @crepererum @xhochy

Reply:
I think you're right:

```python
>>> import pandas as pd
>>> import pyarrow as pa
>>> df = pd.DataFrame({"x": [pd.Timestamp("now")]})
>>> pa.Schema.from_pandas(df)
x: timestamp[ns]
metadata
--------
OrderedDict([(b'pandas',
              b'{"index_columns": [{"kind": "range", "name": null, "start": '
              b'0, "stop": 1, "step": 1}], "column_indexes": [{"name": null,'
              b' "field_name": null, "pandas_type": "unicode", "numpy_type":'
              b' "object", "metadata": {"encoding": "UTF-8"}}], "columns": ['
              b'{"name": "x", "field_name": "x", "pandas_type": "datetime", '
              b'"numpy_type": "datetime64[ns]", "metadata": null}], "creator'
              b'": {"library": "pyarrow", "version": "0.13.0"}, "pandas_vers'
              b'ion": null}')])
```

Reply:
I believe it is `us` because we use `coerce_timestamps="us"` in the write path but honestly, I can't remember why.